manifest.yaml

```diff
@@ -592,7 +592,8 @@
 - Name: Xamarin.Build.Download
   BuildScript: ./Util/Xamarin.Build.Download/build.cake
   TriggerPaths: [ Util/Xamarin.Build.Download ]
-  MacBuildTargets: [ nuget ]
+  MacBuildTargets: [ ci ]
+  WindowsBuildTargets: [ ci ]
```

Comment:
I like this pattern of adding both the Windows and Mac agents in. Should start showing any cross-platform issues before it even gets to the gate.